DSpace/DSpace
# DS-8759: check that zip file exists and has correct MIME type; clean up imports directory on failed import

## References
* Fixes #8759 

## Description
This PR does three things:
- Adds check to make sure that zip file exists and has the correct MIME type
- Removes the temporary zip file if an SAF import process fails in the UI.
- Changes the SAF import process in the UI to create a temporary work directory the same way it's done when importing via the command line so that the "importSAF" work folder is not deleted.

## Instructions for Reviewers
Test one
1. Run an SAF import on the command line and specify a filename of a file that does not exist (with the -z flag)
2. Observe the error message that the file is missing

Test two
1. Run an SAF import on the command line and specify a file that is NOT a zip file (with the -z flag)
2. Observe the error message about incorrect MIME type

Test three
1. Perform an SAF import in the UI with an import file that has metadata problems (e.g. [saf-bitstreams-invalid.zip](https://github.com/DSpace/DSpace/files/11579263/saf-bitstreams-invalid.zip)), so that it results in a failed import
2. Check /dspace/imports folder and note that the temporary zip file is gone, but the empty /dspace/imports/importSAF folder still exists

## Checklist
